api/src/test/java/org/openmrs/module/openconceptlab/client/OclClientTest.java

```diff
@@ -132,4 +132,13 @@ public void parseDateFromPath_shouldParseDate() throws Exception {
 		
 		assertThat(date, is(calendar.getTime()));
 	}
+
+	@Test
+	public void getUrlVersion_shouldGetUrlVersion() throws IOException {

```

Comment:
Can we add another test for what the expected return value should be for cases dealing with the latest released version?

Reply:
Noted